packages/design-system/src/components/primitives/numeric-gesture-control.ts

```diff
@@ -39,7 +39,7 @@ type NumericScrubState = {
   offset: number;
   velocity: number;
   direction: string;
-  timeout: unknown;
+  timeout?: ReturnType<typeof window.setTimeout>;
```

Comment:
thats timerId

Reply:
timeout is the argument you pass to setTimeout, what you receive is an id for that timer